### DISCUSSION TITLE
How to post empty array for multi select

### DISCUSSION THREAD

**Discussion by ak-14** [orchidjs/tom-select#182] [Q&A]
I am using Tomselect with `<select multiple>` element in a form. When I deselect all items and submit the form i expect an empty array to be submitted but instead no value is submitted for the select. Is there a way to configure Tomselect to post an empty array when no items are selected?

Thanks

**Comment by Mepcuk** [comment]
@ak-14 mayne need this field send optional? Can you describe your use case?
